src/pages/variants/ConsequencesCell.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
We should reconsider using that pattern.

see . https://github.com/facebook/create-react-app/pull/8177

This is not a blocker since we use it elsewhere. But we should avoid it in next PR.

Please for the moment use { FC } instead. It will easy the future refactoring since it's how we use it normally.